vue2/src/views/About.vue

```diff
@@ -1,5 +1,44 @@
 <template>
-  <div class="about">
-    <h1>This is an about page</h1>
+  <div class="about" >
+		<div>
+			<h3>以下にお答えください</h3>
+			<div>現在生命保険に加入されていますか？</div>
+			<label><input @click="openQuestion2"  type="radio" name="1"  >はい</label>
+			<label><input @click="openQuestion2"  type="radio" name="1"  >いいえ</label>
+			<div v-if="question2">現在入院中ですか。または、最近3ヵ月以内に医師の診察・検査の結果、入院・手術を進められことはありますか？</div>
+			<label v-if="question2"><input v-if="question2" @click="openQuestion3" type="radio" name="2"  >はい</label>
+			<label v-if="question2"><input v-if="question2" @click="openQuestion3" type="radio" name="2"  >いいえ</label>
+			<div v-if="question3">過去5年以内に、病気やけがで、手術を受けたことまたは継続して7日以上の入院をしたことがありますか？</div>
+			<label v-if="question3"><input v-if="question3" type="radio" name="3"  >はい</label>
+			<label v-if="question3"><input v-if="question3" type="radio" name="3"  >いいえ</label>
+		</div>
+		
+		<button @click="beforePage" >＜  前へ戻る</button>
+		<button @click="nextPage" >次へ進む   ＞</button>
   </div>
 </template>
+
+<script>
+export default {
+	data(){
+		return{
+			question2:false,
+			question3:false
+		}
+	},
+  methods: {
+		beforePage () {
+      this.$router.push('/')
+    },
+    nextPage () {
+      this.$router.push('/3')
+    },
+		openQuestion2(){
```

Comment:
インデントがずれていますので、修正しましょう。
```suggestion
openQuestion2(){
```